### PR TITLE
Mission - explain the radius and path.

### PR DESCRIPTION
**Pull request by hamishwillee**
Hi @bresch 

This just tries to capture your helpful response on waypoint trajectory on slack earlier in the week.

I haven't tested that this properly renders the equation (as reproduced below) but I will do so after this merged, if the other text is OK.
image

### CODE REVIEW

File: en/flight_modes/mission.md [hunk 1]
```diff
@@ -122,4 +122,18 @@ Note:
...
+PX4 expects to follow a straight line from the previous waypoint to the current target (it does not plan any other kind of path between waypoints - if you need one you can simulate this by adding additional waypoints).
+
+MC vehicles will change the *speed* when approach/leaving a waypoint based on whether it uses [slew-rate](../config_mc/mc_slew_rate_type_trajectory.md#mission-mode) or [jerk-limited](
+../config_mc/mc_jerk_limited_type_trajectory.md#auto-mode) tuning.
 
+Vehicles switch to the next waypoint as soon as they enter the acceptance radius. 
+- For MC this radius is defined by [NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD)
+- For FW the radius is defined by the "L1 distance".
+  - The L1 distance is computed from two parameters: [L1_DAMPING](../advanced_config/parameter_reference.md#L1_DAMPING) and [L1_PERIOD](../advanced_config/parameter_reference.md#L1_PERIOD), and the current ground speed.
+  - By default, it's about 60-70 meters.
```
Comment (bresch) [inline en/flight_modes/mission.md]: It's actually 70m (I was too lazy to compute it when I gave the answer in slack...)

https://www.wolframalpha.com/input/?i=20*15*0.75%2Fpi

Comment (hamishwillee) [reply en/flight_modes/mission.md]: Thanks!

File: en/flight_modes/mission.md [hunk 2]
```diff
@@ -122,4 +122,18 @@ Note:
...
+MC vehicles will change the *speed* when approach/leaving a waypoint based on whether it uses [slew-rate](../config_mc/mc_slew_rate_type_trajectory.md#mission-mode) or [jerk-limited](
+../config_mc/mc_jerk_limited_type_trajectory.md#auto-mode) tuning.
 
+Vehicles switch to the next waypoint as soon as they enter the acceptance radius. 
+- For MC this radius is defined by [NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD)
+- For FW the radius is defined by the "L1 distance".
+  - The L1 distance is computed from two parameters: [L1_DAMPING](../advanced_config/parameter_reference.md#L1_DAMPING) and [L1_PERIOD](../advanced_config/parameter_reference.md#L1_PERIOD), and the current ground speed.
+  - By default, it's about 70 meters.
+  - The equation is:
+    $$ L_{1_{distance}}=\frac{1}{\pi}L_{1_{damping}}L_{1_{period}}\left \| \vec{v}_{{xy}_{ground}} \right \|) $$
```
Comment (bresch) [inline en/flight_modes/mission.md]: @hamishwillee https://docs.px4.io/master/en/flight_modes/mission.html#inter-waypoint-trajectory
There is a closing bracket that shouldn't be there at the end of the equation :|

Comment (hamishwillee) [reply en/flight_modes/mission.md]: Thanks @bresch! I will fix. Just testing other rendering issues with this equation (as we speak).

Comment (hamishwillee) [reply en/flight_modes/mission.md]: Done, rendering now working. Might take a few mins to propagate. Thanks for the review!

**Pull request merged**